### DISCUSSION TITLE
Will osu!lazer ever get a big performance update for mobile?

### DISCUSSION THREAD

**Discussion by SpyrosTheBoss** [ppy/osu#24592] [Q&A] [answered]
Asking cause I got lazer running at 20 fps on  Android on Raspberry Pi 4.

**Comment by peppy** [comment, marked as answer]
Performance and optimisation considerations are an ongoing effort. Especially on mobile devices, where the hardware can be of varying quality, it may be impossible to run well on some devices. Please be patient and wait for future optimisations, or upgrade your device.

**Reply by bdach** [reply]
And if I'm honest android on a raspberry pi is not a performance target _any_ game made within the last 15 years is going to be aiming for, consciously or not. The fact that it even works should be considered a miracle. The thing doesn't even seem to _have a GPU_ as far as I can tell.
